compiler/il/OMRIL.cpp

```diff
@@ -596,10 +596,6 @@ OMR::IL::opCodeForCorrespondingIndirectLoad(TR::ILOpCodes loadOpCode)
       case TR::aloadi: return TR::astorei;
       case TR::vloadi: return TR::vstorei;
       case TR::cloadi: return TR::cstorei;
-      case TR::buloadi: return TR::bustorei;
-      case TR::iuload: return TR::iustore;
-      case TR::iuloadi: return TR::iustorei;
-      case TR::luloadi: return TR::lustorei;
```

Comment:
Rather than deleteing the cases I would suggest mapping the buloadi to bstorei etc. The cases can be removed when the opcodes are removed. If you remove the cases things may break unexpectedly if one of the opcodes can find their way in.

Reply:
The purpose of this change is to stop generating the unsigned deprecated Store opcodes on the `return` side. Since the unsigned Load opcodes in the corresponding switch case expressions are also stopping generating in this same PR, would it be safe to just delete these cases?

Reply:
Fixed in https://github.com/eclipse/omr/commit/75af263cf67d4465ea92025ed663d51a0ab12495